xdsl/dialects/comb.py

```diff
@@ -153,21 +227,115 @@ class XorOp(VariadicCombOperation):
     name = "comb.xor"
 
 
+@dataclass
```

Comment:
Remove this annotation? If you inherit from `IRDLOperation` it's already annotated with it.